week6/seojin/.gitignore

```
@@ -0,0 +1,3 @@
+.DS_Store
+*/node_modules
```

node_modules가 올라왔습니다! ㅠㅠ 
```suggestion
node_modules/
```
이렇게 바꿔주세요!! 잘못 알려드려서 죄송합니다 이렇게 해야 node_modules가 안올라온다고 합니다.
이미 올라온 node_modules는 다시 제거해주세요 ㅠㅠ